public/main/webservices/api/v2.php

```diff
@@ -45,6 +45,27 @@
     }
 
     switch ($action) {
+
+        case Rest::GET_HOMEWORK:
+
+            $rest= new Rest($username,$apiKey);
+            $idCourse=$_REQUEST['c_id'];
```

Comment:
Don't use `$_REQUEST['c_id']`. The request param to use is `course` and it's set in line 33, like with session.

Please, check `Rest::getCourseDocuments` for reference.